<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsDLWindowDockWidgetBase</class>
 <widget class="QgsDockWidget" name="QgsDLWindowDockWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map Canvas</string>
  </property>
  <widget class="QWidget" name="mContents">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QToolBar" name="mToolbar">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="floatable">
       <bool>false</bool>
      </property>
      <addaction name="action_tiqudianlixian"/>
      <addaction name="m_drawlieonprofile"/>
      <addaction name="m_selectiononprofile"/>
      <addaction name="m_drawlieonprofile_2"/>
      <addaction name="mActionSaveEdits"/>
      <addaction name="mActionViewInTable"/>
      <addaction name="separator"/>
      <addaction name="mActionPan"/>
      <addaction name="showshaderparameter"/>
      <addaction name="mActionMeasureAngle"/>
      <addaction name="mActionMeasureDistance"/>
      <addaction name="mActionTurnLeft"/>
      <addaction name="mActionTurnRight"/>
      <addaction name="actionmActionUpCenter"/>
      <addaction name="actionmActionDownCenter"/>
      <addaction name="separator"/>
      <addaction name="action_xiubudianlixian"/>
      <addaction name="mActionPickPoints"/>
      <addaction name="mActionBrush"/>
      <addaction name="mActionSinglePointPen"/>
      <addaction name="separator"/>
      <addaction name="action_chongjiandianlixian"/>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
   </layout>
  </widget>
  <action name="mActionSinglePointPen">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/cursors/mSampler.svg</normaloff>:/images/themes/default/cursors/mSampler.svg</iconset>
   </property>
   <property name="text">
    <string>刻刀工具</string>
   </property>
   <property name="toolTip">
    <string>刻刀工具</string>
   </property>
  </action>
  <action name="mActionBrush">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/propertyicons/rendering.svg</normaloff>:/images/themes/default/propertyicons/rendering.svg</iconset>
   </property>
   <property name="text">
    <string>刷子工具</string>
   </property>
   <property name="toolTip">
    <string>刷子</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="mActionPickPoints">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/propertyicons/render.svg</normaloff>:/images/themes/default/propertyicons/render.svg</iconset>
   </property>
   <property name="text">
    <string>拾取点，并显示在表格中</string>
   </property>
   <property name="toolTip">
    <string>拾取点，并显示在表格中</string>
   </property>
  </action>
  <action name="m_selectiononprofile">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/cc/square.png</normaloff>:/images/cc/square.png</iconset>
   </property>
   <property name="text">
    <string>在剖面图上框选点云</string>
   </property>
  </action>
  <action name="m_drawlieonprofile">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/checks/MissingVertex.svg</normaloff>:/images/themes/default/checks/MissingVertex.svg</iconset>
   </property>
   <property name="text">
    <string>分类上侧点</string>
   </property>
  </action>
  <action name="m_drawlieonprofile_2">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/cadtools/perpendicular.svg</normaloff>:/images/themes/default/cadtools/perpendicular.svg</iconset>
   </property>
   <property name="text">
    <string>分类下侧点</string>
   </property>
  </action>
  <action name="mActionSaveEdits">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionSaveAllEdits.svg</normaloff>:/images/themes/default/mActionSaveAllEdits.svg</iconset>
   </property>
   <property name="text">
    <string>保存分类结果</string>
   </property>
   <property name="toolTip">
    <string>保存分类结果</string>
   </property>
  </action>
  <action name="mActionViewInTable">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionOpenTable.svg</normaloff>:/images/themes/default/mActionOpenTable.svg</iconset>
   </property>
   <property name="text">
    <string>Zoom map to the selected rows</string>
   </property>
   <property name="toolTip">
    <string>Zoom map to the selected rows (Ctrl+J)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+J</string>
   </property>
  </action>
  <action name="showshaderparameter">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/cc/ccGear.png</normaloff>:/images/cc/ccGear.png</iconset>
   </property>
   <property name="text">
    <string>shaderparameter</string>
   </property>
  </action>
  <action name="actionmAction_Z">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/icons/z.png</normaloff>:/images/icons/z.png</iconset>
   </property>
   <property name="text">
    <string>Assign_Z</string>
   </property>
  </action>
  <action name="mActionMeasureAngle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionMeasureAngle.svg</normaloff>:/images/themes/default/mActionMeasureAngle.svg</iconset>
   </property>
   <property name="text">
    <string>Measure Angle</string>
   </property>
  </action>
  <action name="mActionMeasureDistance">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionMeasure.svg</normaloff>:/images/themes/default/mActionMeasure.svg</iconset>
   </property>
   <property name="text">
    <string>测量距离</string>
   </property>
   <property name="toolTip">
    <string>MeasureDistance</string>
   </property>
  </action>
  <action name="mActionTurnLeft">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionArrowLeft.svg</normaloff>:/images/themes/default/mActionArrowLeft.svg</iconset>
   </property>
   <property name="text">
    <string>左旋</string>
   </property>
  </action>
  <action name="mActionTurnRight">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionArrowRight.svg</normaloff>:/images/themes/default/mActionArrowRight.svg</iconset>
   </property>
   <property name="text">
    <string>右旋转</string>
   </property>
  </action>
  <action name="mActionPan">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionPan.svg</normaloff>:/images/themes/default/mActionPan.svg</iconset>
   </property>
   <property name="text">
    <string>Pan Map</string>
   </property>
  </action>
  <action name="actionmActionUpCenter">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionArrowUp.svg</normaloff>:/images/themes/default/mActionArrowUp.svg</iconset>
   </property>
   <property name="text">
    <string>mActionUpCenter</string>
   </property>
  </action>
  <action name="actionmActionDownCenter">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionArrowDown.svg</normaloff>:/images/themes/default/mActionArrowDown.svg</iconset>
   </property>
   <property name="text">
    <string>mActionDownCenter</string>
   </property>
   <property name="toolTip">
    <string>mActionDownCenter</string>
   </property>
  </action>
  <action name="action_tiqudianlixian">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/cc/mapIcon.png</normaloff>:/images/cc/mapIcon.png</iconset>
   </property>
   <property name="text">
    <string>电力线提取</string>
   </property>
  </action>
  <action name="action_chongjiandianlixian">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/themes/default/algorithms/mAlgorithmFuzzifyGaussian.svg</normaloff>:/images/themes/default/algorithms/mAlgorithmFuzzifyGaussian.svg</iconset>
   </property>
   <property name="text">
    <string>电力线重建</string>
   </property>
  </action>
  <action name="action_xiubudianlixian">
   <property name="icon">
    <iconset resource="../../images/images.qrc">
     <normaloff>:/images/north_arrows/gpsarrow2.svg</normaloff>:/images/north_arrows/gpsarrow2.svg</iconset>
   </property>
   <property name="text">
    <string>模型离散化</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDockWidget</class>
   <extends>QDockWidget</extends>
   <header>qgsdockwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
  <include location="../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
